test_scripts/API/SendLocation/common_send_location.lua

```diff
@@ -0,0 +1,283 @@
+---------------------------------------------------------------------------------------------------
+-- RC common module
```

Comment:
@Itileda Please rename module

Reply:
@aderiabin , fixed in 9ae38a4203ca7367bfbafed1628d5cb1a9e9fe0d